plugins/modules/remote_management/redfish/ilo_redfish_config.py

```diff
@@ -167,8 +168,9 @@ def main():
 
         for command in command_list:
             result[command] = dispatch[command](mgr_attributes)
+            changed = result[command]['changed']
```

Comment:
```suggestion
            changed |= result[command]['changed']
```
Otherwise you have a problem when the user requests multiple commands, but not all of them change something.

Reply:
Also `changed` is not always present in the result dict. See for example https://github.com/ansible-collections/community.general/pull/3740/files#diff-360533a0c89cbb4164a90103a5e08a3396822855d455d0a5ccec9b27c3aad1bbR118

Reply:
Made the changes.

Reply:
The line I linked to was **one example out of quite a few**. You should really fix the calling code to not rely on `changed` being present.